LibVLCSharp/Shared/MediaPlayer.cs

```diff
@@ -10,621 +10,618 @@ public class MediaPlayer : Internal
                 EntryPoint = "libvlc_media_player_event_manager")]
             internal static extern IntPtr LibVLCMediaPlayerEventManager(IntPtr mediaPlayer);
 
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_is_playing")]
             internal static extern int LibVLCMediaPlayerIsPlaying(IntPtr mediaPlayer);
 
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_play")]
             internal static extern int LibVLCMediaPlayerPlay(IntPtr mediaPlayer);
 
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_set_pause")]
             internal static extern void LibVLCMediaPlayerSetPause(IntPtr mediaPlayer, bool pause);
 
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_pause")]
             internal static extern void LibVLCMediaPlayerPause(IntPtr mediaPlayer);
 
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_stop")]
             internal static extern void LibVLCMediaPlayerStop(IntPtr mediaPlayer);
-#if COCOA
-            [SuppressUnmanagedCodeSecurity]
+
+#if COCOA || NET40
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_set_nsobject")]
             internal static extern void LibVLCMediaPlayerSetNsobject(IntPtr mediaPlayer, IntPtr drawable);
 
-            [SuppressUnmanagedCodeSecurity]
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_get_nsobject")]
             internal static extern IntPtr LibVLCMediaPlayerGetNsobject(IntPtr mediaPlayer);
 #endif
-            [SuppressUnmanagedCodeSecurity]
+
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_set_xwindow")]
             internal static extern void LibVLCMediaPlayerSetXwindow(IntPtr mediaPlayer, uint drawable);
 
-            [SuppressUnmanagedCodeSecurity]
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl,
                 EntryPoint = "libvlc_media_player_get_xwindow")]
             internal static extern uint LibVLCMediaPlayerGetXwindow(IntPtr mediaPlayer);
-#if DESKTOP
-            [SuppressUnmanagedCodeSecurity]
+
+#if NETSTANDARD2_0 || NET40
```

Comment:
NETSTANDARD1_1 can't do that?

Reply:
if by _that_ you mean `SuppressUnmanagedCodeSecurity`, nope.